### PR TITLE
 pr for no reason because you said so #2

### PR DESCRIPTION
**Pull request by djhx3**
commit to main done before which i dont bother to change now because i have to do something actually beneficial

### CODE REVIEW

File: DSA/README.md
Comment (Himesh-Kundal) [inline DSA/README.md:1]: instead of just dumping a link, make a anchor tag and write the title. do you know html?

Comment (djhx3) [reply DSA/README.md:1]: bro i thought you said to just pr for no reason
what exactly do you want then, tell, I will do then pr

Comment (Himesh-Kundal) [reply DSA/README.md:1]: No need to make a new pr, just make commits and they will show in this pr, just watch the fkin tutorial for gods sake. Make a beautiful redame for resources of dsa.

Comment (djhx3) [reply DSA/README.md:1]: Fine, but just for clarity, you want me to watch tutorial on pr or readme or what

**Pull request merged**